models/article-model.js

```diff
@@ -76,17 +73,14 @@ exports.getArticlesQuery = ((next, topic='') => {
     }
     else {
         // articles not filtered by topic
-        return db.query(`SELECT a.article_id, a.title, a.topic, a.author, a.    created_at, a.votes, a.article_img_url, count(c.article_id) as comment_count
+        return db.query(`SELECT a.article_id, a.title, a.topic, a.author, a.    created_at, a.votes, a.article_img_url, CAST(COUNT(a.article_id) AS INTEGER) AS comment_count
         FROM articles a
         JOIN comments c
             ON a.article_id = c.article_id
         GROUP BY a.article_id
         ORDER BY a.created_at DESC;`)
 
         .then(({rows}) =>{
-            rows.map((row) => {
-                row.comment_count = Number(row.comment_count);
-            })
```

Comment:
Good removal here - the refactored version make a lot more sense and would be more efficient (which is quite important especially when it comes to large projects)